javascript/javascript1/week1/task_1.js

```diff
@@ -0,0 +1,6 @@
+//future age calculator
+
+let yearOfBirth = 1981;
+let yearFuture = 2030;
+const age = yearFuture - yearOfBirth;
+console.log("you will be", age, "years old in", yearFuture);
```

Comment:
another way to combine string and variable https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals